src/condition.cpp

```diff
@@ -1491,14 +1491,13 @@ std::function<int( const T & )> conditional_t<T>::get_get_int( const JsonObject
             return [info]( const T & d ) {
                 std::string var = read_var_value( info, d );
                 if( !var.empty() ) {
-                    return std::stoi( var );
-                } else {
-                    try {
-                        return std::stoi( info.default_val );
-                    } catch( const std::exception & ) {
-                        return 0;
-                    }
+                    // NOLINTNEXTLINE(cert-err34-c)
+                    return std::atoi( var.c_str() );
```

Comment:
According to https://en.cppreference.com/w/cpp/string/byte/atoi this triggers UB if the result would be out of range.

Maybe std::strtol is suitable if you need something non-throwing?

Reply:
This part is actually me updating it to the latest version in master, I didn't make this change someone else did.